src/clj/puppetlabs/http/client/async.clj

```diff
@@ -175,12 +176,13 @@
   ([opts :- common/RawUserRequestOptions
     callback :- common/ResponseCallbackFn
     client :- HttpAsyncClient]
-    (request-with-client opts callback client nil nil))
+    (request-with-client opts callback client nil nil true))
   ([opts :- common/RawUserRequestOptions
     callback :- common/ResponseCallbackFn
     client :- HttpAsyncClient
     metric-registry :- (schema/maybe MetricRegistry)
-    metric-namespace :- (schema/maybe schema/Str)]
+    metric-namespace :- (schema/maybe schema/Str)
+    use-url-metrics :- (schema/maybe schema/Bool)]
```

Comment:
If above is `true` is it possible for this to be nil / should we make it required?